judo-runtime-core-dao-core/src/main/java/hu/blackbelt/judo/runtime/core/dao/core/processors/DeletePayloadDaoProcessor.java

```diff
@@ -184,7 +182,7 @@ void collectStaments(EClass entityType,
                                 .build());
                     }
                     if (r.getReference().getEOpposite() != null && AsmUtils.annotatedAsTrue(r.getReference().getEOpposite(), "reverseCascadeDelete")) {
-                        collectStaments(r.getReference().getEReferenceType(),
```

Comment:
> collectStaments

If possible this typo should be fixed as well, If I'm not mistaken, this should be `collectStatements`

Reply:
[0382708](https://github.com/BlackBeltTechnology/judo-runtime-core/pull/201/commits/0382708dba99d21fc499ed64fdba464108421a03)